ansible/roles/cri/tasks/containerd.yml

```diff
@@ -0,0 +1,76 @@
+# Instructions: https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd
+---
+- name: Copy containerd.conf
+  template:
+    src: containerd.conf.j2
+    dest: /etc/modules-load.d/containerd.conf
+
+- name: Load overlay
+  modprobe:
+    name: overlay
+    state: present
+
+- name: Load br_netfilter
+  modprobe:
+     name: br_netfilter
+     state: present
+
+- name: Set bridge-nf-call-iptables
```

Comment:
We make some of these sysctl commands in the common role - I think Docker might also require some of these, if that is the case we should remove the ones from common and move them to their respect CRI, however, if they're the same perhaps we can create a new role that runs these after the CRI if they are indeed the same for both.

Reply:
According to https://kubernetes.io/docs/setup/production-environment/container-runtimes/#docker, seems like docker doesn't need these sysctl calls, should I just remove these calls from common role? 
But let's assume they both need these commands, shouldn't we run these commands before the CRI?(Cause you said "if they're the same perhaps we can create a new role that runs these after the CRI if they are indeed the same for both.")